src/ImageSharp/Formats/Png/PngHeader.cs

```diff
@@ -116,7 +116,7 @@ public void Validate()
         /// <param name="buffer">The buffer to write to.</param>
         public void WriteTo(Span<byte> buffer)
         {
-            BinaryPrimitives.WriteInt32BigEndian(buffer.Slice(0, 4), this.Width);
+            BinaryPrimitives.WriteInt32BigEndian(buffer[..4], this.Width);
             BinaryPrimitives.WriteInt32BigEndian(buffer.Slice(4, 4), this.Height);
```

Comment:
Did you keep this a `Slice` on purpose?

Reply:
It wasn't suggested by the IDE. There's situations where is wont be due to performance differences. Maybe this is one of them?